test/third_party_tests/test_native_executor.py

```diff
@@ -169,6 +169,22 @@ def test_should_run_query_in_postgres(self):
         self._raise_error_on_multiple_creation()
         self._raise_error_on_invalid_connection()
 
+    def test_should_run_query_in_mariadb(self):
+        # Create database.
+        params = {
+            "user": "eva",
+            "password": "password",
+            "database": "evadb",
```

Comment:
Same question as the previous discussion. Does your integration test pass without `host` and `port` here?

Reply:
Yes the test is passing locally